processor/policysamplingprocessor/config.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policysamplingprocessor
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// PolicyType indicates the type of sampling policy.
+type PolicyType string
```

Comment:
Thanks for the review! This part is inherited from the current tail-based sampling, but you are right. In fact, I think we could discuss making this go in another direction: instead of one "policysamplingprocessor", we could be having two or three processors: `ratelimitingprocessor` (`rate_limiting` from this PR) and `attributesamplingprocessor` (string and int policies from this PR). The `always` sampling would just disappear, as it's not really useful IMO.

What do you think?